R/rplot.R

```diff
@@ -0,0 +1,94 @@
+#' @title Rasterly plot
+#' @name rplot
+#' @description \code{rplot} is created to generate \code{rasterly} plot quickly but with base 
+#' \link{plot} design. It is convenient but lack of flexibility and \link{rasterly} is highly
+#' recommended.
+#' @param x,y Coordinates x, y for the plot. 
+#' @param ... Other aesthetics to pass through, like \code{color}, \code{size} and \code{on}.
+#' @export
+#' @seealso \link{rasterly} \link{rasterly_points}
+rplot <- function(x, y = NULL, ...) {
+  UseMethod("rplot")
+}
+
+#' @inherit rplot
+#' @rdname rplot
+#' @param plot_width Integer. The width of the image to plot; must be a positive integer. A higher value indicates a higher resolution.
+#' @param plot_height Integer. The height of the image to plot; must be a positive integer. A higher value indicates a higher resolution.
+#' @param x_range Vector of type numeric. The range of \code{x}; it can be used to clip the image. For larger datasets, providing \code{x_range}
+#' may result in improved performance.
+#' @param y_range Vector of type numeric. The range of \code{y}; it can be used to clip the image. For larger datasets, providing \code{y_range}
+#' may result in improved performance.
+#' @param background Character. The background color of the image to plot.
+#' @param reduction_func Function. A reduction function is used to aggregate data points into their pixel representations. Currently
+#' supported reduction operators are \code{sum}, \code{any}, \code{mean}, \code{m2}, \code{first}, \code{last}, \code{min} and \code{max}. Default is \code{sum}. See details.
+#' @param layout Character. The method used to generate layouts for multiple images. The default is \code{weighted}. Useful for categorical
+#' data (i.e. "color" is provided via \code{aes()}). \code{weighted} specifies that the final raster should be a weighted combination of each
+#' (categorical) aggregation matrix. Conversely, \code{cover} indicates that the afterwards objects will be drawn on top of 
+#' the previous ones.
+#' @param glyph Character. Currently, only "circle" and "square" are supported; as the \code{size} of the pixels increases, how should they
```

Comment:
Maybe I'm not using a sufficiently large enough dataset, but with a 1 million point subset of `ridesDf`, I see no difference when changing the parameter from "circle" to "square". Does this function perhaps need an extra "size" param for this to work?

Reply:
size should be numerical value, 1,2,3,4..., etc.

Reply:
I see, it does work, and that's a great feature to have. I would suggest defining it within the `rplot` function as well as within the documentation so a user would know that it's available.

Reply:
How about putting them in documentation `Details`? Since I checked documentation of function `plot.default`, the aesthetics are passed through via ...